Translator/Header/Reader.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+#ifndef READER_H
```

Comment:
You can remove the #ifndef guards since #pragma once already prevents multiple inclusions